dds/idl/marshal_generator.cpp

```diff
@@ -1739,9 +2183,11 @@ namespace {
         if (accessor) {
           local.erase(local.size() - 2);
         }
-        intro += "  " + getArrayForany(pre.c_str(), name.c_str(), tdname) + '\n';
+        //TODO: Was this redundant call fixed
+        intro += "  " + getArrayForany(pre.c_str(), name.c_str(), tdname, "_sc") + '\n';
+        local += "_sc";
```

Comment:
@mitza-oci @lij-oci The redefinition error this is causing is keeping me from building the try construct test, which I'd really like to be able to do to make sure I'm not breaking what @ClaytonCalabrese is doing. This has to be fixed. I think my previous proposed solution of turning intro strings into a set or something like this is still valid.

Reply:
To approximate my solution I put a [workaround](https://github.com/iguessthislldo/OpenDDS/commit/0c69a517b2a0d2e8bc9e05bf1824aa601dbc9453) in #1894 just to get it to build.

Reply:
Please add a test case in `tests/DCPS/Compiler` so we can fix it outside of xtypes